std_msgs/Header header
  uint32 seq
  time stamp
  string frame_id
bool status_left 
bool status_right 